svm-feature-set/src/lib.rs

```diff
@@ -0,0 +1,82 @@
+#[derive(Clone, Copy, Default)]
+pub struct SVMFeatureSet {
```

Comment:
is this a separate pr backport or merge conflict resolution error?

Reply:
Ah yes, merge conflict resolution error. Will remove it.